packages/cli/ete-tests/src/tests/ir/__snapshots__/ir.test.ts.snap

```diff
@@ -30011,6 +30035,7 @@ exports[`ir {"name":"simple"} 1`] = `
                         },
                         {
                             "exampleType": "userProvided",
+                            "id": null,
```

Comment:
@armandobelardo im guessing we want ids here as well

Reply:
I don't think we want to generate for user provided right? Cause then it changes with every generation

So I made it exposed in the yaml schema and a user can specify the id